.github/workflows/promote-to-beta.yml

```diff
@@ -33,18 +33,22 @@ jobs:
       with:
         username: ${{ secrets.DOCKERHUB_USERNAME }}
         password: ${{ secrets.DOCKERHUB_TOKEN }}
-    - name: Promote images from develop to beta
+        

```

Comment:
do we need to set up docker buildx beforehand or will it just work with the docker-login action?

Reply:
We could add it, but since we only copy an image and dont build one, I think we dont need it, because it just [creates an builder](https://github.com/docker/setup-buildx-action#about)  and in my test it worked without it

Reply:
okay cool! Good to know!